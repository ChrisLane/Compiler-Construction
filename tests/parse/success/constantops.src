main() {
    var x = 10 + 10;
    x = 20 * 10;
}
