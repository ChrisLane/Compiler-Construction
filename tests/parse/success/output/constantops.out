
main (  ) { 
    New ( "x" = Operator ( Plus, Const 10 , Const 10 ) ) In { 
        Asg ( Identifier "x" := Operator ( Times, Const 20 , Const 10 ) ) 
    } 
}

